lib/Controller/PageController.php

```diff
@@ -332,12 +341,23 @@ public function recording(string $token): Response {
 	}
 
 	/**
-	 * @param string $token
-	 * @param string $password
 	 * @return TemplateResponse|RedirectResponse
 	 * @throws HintException
 	 */
-	protected function guestEnterRoom(string $token, string $password): Response {
+	protected function guestEnterRoom(
+		string $token,
+		string $password,
+		string $email,
+		#[SensitiveParameter]
+		string $accessToken,
+	): Response {
+		if ($email && $accessToken) {
+			return $this->invitedEmail(
+				$token,
+				$email,
+				$accessToken,
+			);
+		}
 		try {
```

Comment:
Same link without queries works as public room, but that's for step 2, I guess

Reply:
Well yes, that's the current behaviour atm